src/Bridge/Symfony/Routing/CachedRouteNameResolver.php

```diff
@@ -39,7 +39,8 @@ public function __construct(CacheItemPoolInterface $cacheItemPool, RouteNameReso
      */
     public function getRouteName(string $resourceClass, $operationType /**, array $context = []**/): string
     {
-        $cacheKey = self::CACHE_KEY_PREFIX.md5(serialize([$resourceClass, $operationType]));
+        $context = \func_num_args() > 2 ? func_get_arg(2) : [];
+        $cacheKey = self::CACHE_KEY_PREFIX.md5(serialize([$resourceClass, $operationType, $context['subresource_resources'] ?? null]));
```

Comment:
We have to add a test!

Reply:
RemindMe! 10 days (I wish there was such a bot here)